app/Http/Controllers/API/ConferencesController.php

```diff
@@ -7,24 +7,17 @@
 use Carbon\Carbon;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
+use App\Http\Requests\StoreConferenceRequest;
 
 class ConferencesController extends Controller
 {
-    public function store(Request $request)
+    public function store(StoreConferenceRequest $request)
     {
-        $after = (new Carbon($request->input('start_date')))->subDay();
```

Comment:
I'm 100% torn about whether I like how the Form Request cleans the method or whether I hate how it unnecessarily complicates the process of discovering the validation rules on this really simple method.